packages/game-app/src/_shared/dynamicData/apis/retrieveDynamicData.ts

```diff
@@ -0,0 +1,38 @@
+import firebase from 'firebase/app';
+import { DevOpsMaturitiesDoc, FirebaseCollections, GameRolesDoc } from '@pipeline/common';
+import { SelectOption } from '@pipeline/models';
+
+import 'firebase/firestore';
+
+const retrieveSelectOptions = (source: { [key: string]: { [key: string]: string } }, lang: string): SelectOption[] => {
+  return Object.keys(source).map(k => {
+    return {
+      value: k,
+      label: source[k][lang],
+    } as SelectOption;
+  });
+};
+
+export async function executeRetrieveGameRoles(lang = 'en'): Promise<SelectOption[]> {
+  const gameRolesDoc = await firebase
+    .firestore()
+    .doc(`${FirebaseCollections.DynamicData}/${FirebaseCollections.GameRoles}`)
```

Comment:
GameRoles is a document id right? and not a collection. Maybe we can add a separated DynamicData Enum